TestAssets/TestPackages/dotnet-desktop-binding-redirects/dotnet-desktop-binding-redirects.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="5.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="5.0.1" />
```

Comment:
??  What was the story here?

Reply:
There was no 5.0.0 version and nuget warned that it had to use 5.0.1